lib/less/tree/call.js

```diff
@@ -7,6 +7,7 @@ var Node = require("./node"),
 var Call = function (name, args, index, currentFileInfo) {
     this.name = name;
     this.args = args;
+    this.mathOn = name === 'calc' ? false : true;
```

Comment:
No, we can't set it to `false` if the name is not `calc` this will break the `sm: on` mode.
It should only:
```js
if (name === `calc`)
   context.mathOn = false;
```
like I suggested earlier. The way it goes in this commit every non-calc function will aways be evaluated with mathon regardless of the main `-sm` fag.

Reply:
> No, we can't set it to true if the name is not calc - this will break the sm: on mode.

Not so. It still has to pass the strictMath test. Setting to true doesn't override this. Relevant code is:
```js
contexts.Eval.prototype.isMathOn = function () {
    if (!this.mathOn) {
        return false;
    }
    return this.strictMath ? (this.parensStack && this.parensStack.length) : true;
};
```
There are still tests to make sure strict math is working. These changes for `calc()` disable math, but enabling does not override strict math rules.